### DISCUSSION TITLE
Can Robot learn as like how we learn languages?????

### DISCUSSION THREAD

**Discussion by ARULANANTHASAMYS** [community/community#73063] [Programming Help] [answered]
### Body

If you wish to teach your robot, kindly have a glance on the following website. 

robotspython.com

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arienshibani** [comment]
If by robots you mean AI, then they do not learn in the same way humans do. They do not have personal experiences or emotions, and they do not acquire knowledge over time. Instead, they generate responses based on a mixture of licensed data, data created by human trainers, and publicly available data. They dont learn language in the human sense. They process language, based on pre-programmed algorithms and patterns.

**Reply by ARULANANTHASAMYS** [reply, marked as answer]
This answer looks great really. But, the robot must have the data in its hand (i.e, Processor) in order to get some useful applications. The way we as humans can make the robot to understand the things better via different languages like python, c, c++ and Java etc., is a challenging task. Kindly you may use the site to practice this better, https://www.robotspython.com/

**Comment by TariqAlhathloul** [comment]
I think that they can learn but not like humans do.
 for example learn a language for a human is like: Vocabulary Building, Grammar, Practice and Interaction,
for robots is like: Input, Training or process, Feedback.
